src/server/game/Spells/Spell.cpp

```diff
@@ -2448,7 +2448,7 @@ void Spell::DoAllEffectOnTarget(TargetInfo* target)
         // Needs to be called after dealing damage/healing to not remove breaking on damage auras
         DoTriggersOnSpellHit(spellHitTarget, mask);
 
-        // if target is fallged for pvp also flag caster if a player
+        // if target is flagged for pvp also flag caster if a player
         if (unit->IsPvP() && m_caster->GetTypeId() == TYPEID_PLAYER)
             m_caster->ToPlayer()->UpdatePvP(true);
```

Comment:
it's always nice to see an issue debugged, understood and fixed without affecting any other functionality. it takes some more time but it's worth it.
thank you for the fine work and for the fix